src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -65,65 +65,64 @@ const defaultProps = {
     walletTransfer: {},
 };
 
-class TransferBalancePage extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.paymentTypes = [
-            {
-                key: CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT,
-                title: this.props.translate('transferAmountPage.instant'),
-                description: this.props.translate('transferAmountPage.instantSummary', {
-                    rate: this.props.numberFormat(CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.RATE),
-                    minAmount: CurrencyUtils.convertToDisplayString(CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.MINIMUM_FEE),
-                }),
-                icon: Expensicons.Bolt,
-                type: CONST.PAYMENT_METHODS.DEBIT_CARD,
-            },
-            {
-                key: CONST.WALLET.TRANSFER_METHOD_TYPE.ACH,
-                title: this.props.translate('transferAmountPage.ach'),
-                description: this.props.translate('transferAmountPage.achSummary'),
-                icon: Expensicons.Bank,
-                type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
-            },
-        ];
-
-        PaymentMethods.resetWalletTransferData();
-        const selectedAccount = this.getSelectedPaymentMethodAccount();
-
-        // Select the default payment account when page is opened,
-        // so that user can see that preselected on choose transfer account page
-        if (!selectedAccount || !selectedAccount.isDefault) {
-            return;
-        }
-
-        PaymentMethods.saveWalletTransferAccountTypeAndID(selectedAccount.accountType, selectedAccount.methodID);
-    }
+function TransferBalancePage(props) {
+    const paymentTypes = [
+        {
+            key: CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT,
+            title: props.translate('transferAmountPage.instant'),
+            description: props.translate('transferAmountPage.instantSummary', {
+                rate: props.numberFormat(CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.RATE),
+                minAmount: CurrencyUtils.convertToDisplayString(CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.MINIMUM_FEE),
+            }),
+            icon: Expensicons.Bolt,
+            type: CONST.PAYMENT_METHODS.DEBIT_CARD,
+        },
+        {
+            key: CONST.WALLET.TRANSFER_METHOD_TYPE.ACH,
+            title: props.translate('transferAmountPage.ach'),
+            description: props.translate('transferAmountPage.achSummary'),
+            icon: Expensicons.Bank,
+            type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
+        },
+    ];
 
     /**
      * Get the selected/default payment method account for wallet transfer
      * @returns {Object|undefined}
      */
-    getSelectedPaymentMethodAccount() {
-        const paymentMethods = PaymentUtils.formatPaymentMethods(this.props.bankAccountList, this.props.cardList);
+    const getSelectedPaymentMethodAccount = useCallback(() => {
```

Comment:
I _think_ we can probably skip `useCallback()` here according to the latest debate on it 😄

Just inline the function and we will only worry about the optimizations if there's a good reason for it. Or if there's some good reason for it we should explain in a comment what it is. If there is, it wasn't obvious to me.